scoop_config:
  driving_arm_angle: -50   #0
  driving_scoop_angle: -20
  pre_dig_arm_angle: 70
  pre_dig_scoop_angle: 100
  digging_arm_angle: 70
  digging_scoop_angle: 100
  post_dig_arm_angle: 0
  post_dig_scoop_angle: -30
  pre_dump_arm_angle: -45
  pre_dump_scoop_angle: -30
  dumping_arm_angle: -45
  dumping_scoop_angle: 60
  post_dump_arm_angle: -45
  post_dump_scoop_angle: 90